### PR TITLE
Add a new quote :) #13

### PR DESCRIPTION
**Pull request by nicolas-alv3**
Hi! I've added a new quote into quotes.json and my profile into .allcontributorsrc. #13

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -252,6 +252,15 @@
       "contributions": [
         "content"
       ]
+    },
+    {
```
Comment (jayehernandez) [inline .all-contributorsrc:256]: Hope you can remove this, as the bot all-contributors does this automatically when called!

Comment (nicolas-alv3) [reply .all-contributorsrc]: I 've removed changes in all-contributors and added a new quote! :)

**Comment by nicolas-alv3** [comment]
I've made the changes you ask for, hope you find it useful!:)

File: .all-contributorsrc (hunk 1 above)
Comment (jayehernandez) [reply .all-contributorsrc]: Thanks for fixing it!! 👍

**Comment by jayehernandez** [comment]
Thank you for resolving the comments, @nicolas-alv3!

 @all-contributors please add @nicolas-alv3 for content

**Pull request merged**